amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/index.tsx

```diff
@@ -2,23 +2,38 @@ import * as React from 'react';
 import moment from 'moment-timezone';
 
 import ColumnDescEditableText from 'components/TableDetail/ColumnDescEditableText';
+import { GlobalState } from 'ducks/rootReducer';
 import { logClick } from 'ducks/utilMethods';
+import { ToggleRequestAction } from 'ducks/notification/types';
+import { openRequestDescriptionDialog } from 'ducks/notification/reducer';
 import { TableColumn } from 'interfaces';
 
+import { Dropdown, MenuItem } from 'react-bootstrap';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
```

Comment:
nit: You can library imports at the top.

Reply:
+1